src/app/wallet/wallet/send/send.component.ts

```diff
@@ -70,6 +70,7 @@ export class SendComponent {
     this.send.input = 'balance';
     if (this.type === 'balanceTransfer') {
       this.send.toAddress = '';
+      this.send.output = 'blind_balance'
```

Comment:
why this?

Reply:
@kewde yeah if you are on "send payment" and check the `blind` and then moved to "Balance Transfer" in that case `public` is checked on both side which is incorrect.